### PR TITLE
Add hyper training to stats card

### LINKED ISSUE #545: Add hyper training indication
Remember that hyper training doesn't set the IV to 31, it's a seperate bit.

Reaaaaally not sure what the best way to show this is.

### PR DESCRIPTION
**Pull request by YaManicKill**
Fixes #545 

213982e870031c9350cf0ddaf71219c7

### CODE REVIEW

File: client/pokemon/pokemon-full.view.html [hunk 1]
```diff
@@ -171,6 +171,7 @@ <h1 class="summary-toolbar">{{pokemon.parsedNickname}}</h1>
                   <span flex>Stat</span>
                   <span flex>Base</span>
                   <span flex>IVs</span>
+                  <span flex ng-if="pokemon.gen === 7">H/T</span>
```
Comment (not-an-aardvark) [inline client/pokemon/pokemon-full.view.html]: Should we add a "Hyper Training" titletext for this? `H/T` probably isn't obvious to some people.

Comment (Raia) [inline client/pokemon/pokemon-full.view.html]: Yeah, I think we should.

Comment (SnowPhoenix) [inline client/pokemon/pokemon-full.view.html]: I was thinking tooltip if it doesn't look too terrible. That way it's more obvious and I believe that'd keep it usable on mobile, would it not?

Comment (NotRaia) [inline client/pokemon/pokemon-full.view.html]: Mmm, as much as I'd like it to work on mobile, I'd be a bit hesitant to use a tooltip as I don't think they're supposed to be used for static text.

Comment (Raia) [inline client/pokemon/pokemon-full.view.html]: Mmm, as much as I'd like it to work on mobile, I'd be a bit hesitant to use a tooltip as I don't think they're supposed to be used for static text.

Comment (SnowPhoenix) [inline client/pokemon/pokemon-full.view.html]: For clarification, I meant one with a ? icon like we use with the dropdown menus on FlairHQ.

Comment (YaManicKill) [inline client/pokemon/pokemon-full.view.html]: Fwiw, we agreed in the IRC that this was fine with just a titletext. I shall add that.

Comment (YaManicKill) [inline client/pokemon/pokemon-full.view.html]: Done this now.

**Comment by YaManicKill** [comment]
Alright, updated with comments. Anything else? I'll wait for another approval before merging (it can be a re-approval)

**Comment by Raia** [comment]
Looks good.

**Pull request merged**